src/Atmos/Model/AtmosModel.jl

```diff
@@ -174,7 +174,7 @@ end
 
 @inline function flux_diffusive!(atmos::AtmosModel, flux::Grad, state::Vars,
                                  diffusive::Vars, aux::Vars, t::Real)
-  ν, τ = turbulence_tensors(atmos.turbulence, state, diffusive, aux, t)
+  ν, τ, sdiff = turbulence_tensors(atmos.turbulence, state, diffusive, aux, t)
```

Comment:
See comment below

Reply:
I'm currently looking for an effective way of not having this. Since we need a split viscosity for the courant numbers and non split viscosity for the general calculations. "sdiff" is needed  exclusively for the courant calculations.

Reply:
Since it doesn't look like I can avoid having to do a recomputation I think I will write separate function in turbulence to get the split viscosity.